test/e2e/scheduling/taints.go

```diff
@@ -410,10 +410,16 @@ var _ = SIGDescribe("NoExecuteTaintManager Multiple Pods [Serial]", func() {
 
 		By("Starting pods...")
 		nodeName, err := testutils.RunPodAndGetNodeName(cs, pod1, 2*time.Minute)
+		node, err := cs.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
```

Comment:
One suggestion.  You may need to move `framework.ExpectNoError(err)`  from line 419 to line 413 to check the error of `nodeName, err := testutils.RunPodAndGetNodeName(cs, pod1, 2*time.Minute)`